datasets/orcasound/adapt_active_learning_annotations.ipynb

```diff
@@ -0,0 +1,1909 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
```

Comment:
Can you please clear the outputs of the notebook before committing it to git? :) 
https://stackoverflow.com/questions/28908319/how-to-clear-jupyter-notebooks-output-in-all-cells-from-the-linux-terminal
you can also do it directly inside the notebook using the GUI